packages/sit-onyx/src/components/OnyxDataGrid/features/index.ts

```diff
@@ -49,7 +49,7 @@ export type ColumnConfig<TEntry extends DataGridEntry, TTypes> =
  */
 export type NormalizedColumnConfig<TEntry extends DataGridEntry, TTypes = PropertyKey> = {
   key: keyof TEntry;
-  type?: TTypes;
+  type?: TTypes | "string" | "number";
```

Comment:
We should rather change the generic definition, so:

```ts
<TTypes extends PropertyKey = "string" | "number">
```

Reply:
The `TTypes` are the inferred types from features. `"string" | "number"` are hardcoded default types. When defined as default for the generic, they would be overridden by the inferred types from the features. 
This would remove them from the selection.
So the type definition is correct as it currently is defined.

We could however either add a bit more docs or a dedicated type to make it clearer:
```suggestion
  type?: TTypes | DefaultTypes; // where type DefaultTypes = "string" | "number";
```

Reply:
Moved it out into `DefaultSupportedTypes`